### PR TITLE
COMMERCE-9503 - PIM - Thumbnail is not generating while using Headless API with Workflow enabled

### PR DESCRIPTION
**Pull request by pietrobocale**
Resent of #2802 

https://issues.liferay.com/browse/COMMERCE-9503

### CODE REVIEW

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro [hunk 1]
```diff
@@ -270,6 +270,60 @@ definition {
 		com.liferay.poshi.runner.util.JSONCurlUtil.post("${curl}");
 	}
 
+	macro _AddCommerceProductImage {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro:273]: The JSON macro names start in lower case, then continue with camel case, please see the other macros and edit this one: `_AddCommerceProductImage`  to `_addCommerceProductImage`

**Review by baharturk — CHANGES_REQUESTED** [review]
Please see my comment on the macros file. Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro (hunk 1 above)
Comment (pietrobocale) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro]: Fixed

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by pietrobocale** [comment]
ci:test:commerce-funtional

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
ci:forward

**Pull request closed without merging**